templates/includes/media-embed--video.html

```diff
@@ -0,0 +1,60 @@
+<div class="media-embed" style="background: url(/static/images/media-embed-bg.png)">
+    <div class="media-embed__overlay"></div>
+    <div class="media-embed__content-panel">
+        <img class="media-embed__icon" src="/static/images/fontawesomesvgs/video-white.svg" alt="">
+        <p class="media-embed__icon-label" role="presentation" aria-hidden="true">
+            Video
+        </p>
+
+        <h3 class="media-embed__heading">
+            Lorem Ipsum Dolor Sit Amet
+            <span class="sr-only">Video</span>
```

Comment:
As with other comments, I wonder if "Video" should appear before the heading text, possibly followed by a full stop or semi-colon.